web/src/components/feature/calculation_input/CalculationInput.tsx

```diff
@@ -95,6 +96,18 @@ export const CalculationInput = ({
     </Button>
   )
 
+  const savePackage = (newPackage: TPackage) => {
+    let newPackages = [...packages]
```

Comment:
Can this variable be renamed to allPackages or something along those lines? Very hard to see the difference between newPackage and newPackage**s**.

Reply:
True. And the name isn't even correct. Fixed now